website/markdown/docs/usage/best-practices.mdx

```diff
@@ -30,3 +30,12 @@ let target = Target(name: "MyFramework", sources: ["MyFramework/Sources"])
 // Discouraged
 let target = Target(name: "MyFramework", sources: ["MyFramework/**/*.swift"])
 ```
+
+## Using on CI
+
+Adding `tuist` to your project will require you to have installed it in your CI system. You can avoid it by bundling specific version in your repo:
+
+- Define your local version: `tuist local 1.7.1`
+- Bundle it: `tuist bundle 1.7.1`
```

Comment:
The version argument should not be run with the `tuist bundle` command. It is using the version you have specified with the `tuist local` command.